src/main/app/claims/responseModelConverter.ts

```diff
@@ -92,4 +95,7 @@ export class ResponseModelConverter {
     return party
   }
 
+  private static convertWhenDidYouPay (whenDidYouPay: DraftWhenDidYouPay): WhenDidYouPay {
```

Comment:
That will throw an error in case `responseDraft.whenDidYouPay` is undefined and it will be undefined when response > dispute path is selected.